### PR TITLE
Moved aws-common-runtime -> crt

### PR DESCRIPTION
**Pull request by justinboswell**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by singku** [comment]
What's the story behind this move?

**Comment by justinboswell** [comment]
> What's the story behind this move?

shortening submodule paths, which will shorten the recursive dirs that cmake creates, which will allow windows users to have slightly more permissive paths to keep under the 260 MAX_PATH length.

File: .gitmodules [hunk 1]
```diff
@@ -1,30 +1,30 @@
 [submodule "aws-common-runtime/aws-c-common"]
```
Comment (singku) [inline .gitmodules:1]: Didn't change this for readability?

**Review by singku — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (justinboswell) [reply .gitmodules:1]: Didn't change it because it's just a name, and it would break the history.

**Pull request merged**